pkg/util/pipe_windows.go

```diff
@@ -12,19 +12,18 @@ import (
 	"github.com/Microsoft/go-winio"
 )
 
+// WinNamedPipe holds the named pipe configuration
```

Comment:
Are we using the named pipe at all now? After the Auto Discovery/Status JMX changes I'm not sure we need this anymore.... there might still be something? Not sure.